src/sidebar.c

```diff
@@ -191,7 +191,7 @@ void sidebar_update_tag_list(GeanyDocument *doc, gboolean update)
 
 	g_return_if_fail(doc == NULL || doc->is_valid);
```

Comment:
Maybe this should be doc != NULL?

Reply:
erm, no, its or not and, please ignore

Reply:
Yeah, it should be `doc != NULL && doc->is_valid` I think.

Reply:
Nah, I made that mistake first too, but doc == NULL has to keep going to remove the tag tree if there is no doc

Reply:
Ah, OK, got it now too, it's correct.